khmer/_khmer.cc

```diff
@@ -5215,6 +5215,499 @@ static PyObject * hllcounter_merge(khmer_KHLLCounter_Object * me,
     Py_RETURN_NONE;
 }
 
+/********************************
+ * Assembler classes
+ ********************************/
+
+
+typedef struct {
+    PyObject_HEAD
+    LinearAssembler * assembler;
+} khmer_KLinearAssembler_Object;
```

Comment:
So far as I know, it's a holdover. I've just followed the pattern of the existing class defs.